knowledge-base/editor-add-icons.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Add Icons in Editor
+description: How to insert icon tags in Editor?
+type: how-to
+page_title: Add Icons in Editor
+slug: editor-kb-add-icons
+position: 
+tags: editor, blazor, icon, schema, prosemirror
+ticketid: 
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Editor for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+I want to add custom icons in the Editor content. When adding an icon such as `<i class="fa-light fa-envelope">`, the icon's HTML disappears upon saving.
+
+How can I add an `<i>` element inside the Editor?
```

Comment:
This sounds like an alternative question describing the same request. Consider adding "This Knowledge Base article also answers the following question:".